modules/diskq/tests/Makefile.am

```diff
@@ -31,3 +32,10 @@ modules_diskq_tests_test_reliable_backlog_LDADD = $(DISKQ_TEST_LD_ADD)
 modules_diskq_tests_test_reliable_backlog_SOURCES = \
 	modules/diskq/tests/test_reliable_backlog.c \
 	modules/diskq/tests/test_diskq_tools.h
+
```

Comment:
[...]_DEPENDENCIES = $(top_builddir)/modules/diskq/libdisk-buffer.la is missing. 
(Unfortunately this is missing from the old testfiles too.)

Reply:
Good catch, thanks.
I guess it worked before because of this documented behaviour of Automake:
> If _DEPENDENCIES is not supplied, it is computed by Automake. The automatically-assigned value is the contents of _LDADD or _LIBADD, with most configure substitutions, -l, -L, -dlopen and -dlpreopen options removed.

I will add these.